packages/keccak256/src/index.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+
+import { hexValueType, type hexValue } from '@erebos/hex'
+import sha3 from 'js-sha3'
+
+export const hash = (value: Array<number> | Buffer): Array<number> => {
+  return sha3.keccak256.array(value)
+}
+
+export const pubKeyToAddress = (pubKey: Array<number> | Buffer): hexValue => {
```

Comment:
Would be good to handle hex encoded pubkey here too

Reply:
👍 I'll update this.

Reply:
Updated.